README.md

```diff
@@ -44,10 +44,18 @@ Read more about this in the [Lumberjack](#Lumberjack) API section.
 ```javascript
 import Lumberjack from '@americanexpress/lumberjack';
 
+let formatter = null;
+// how you format your message can look like:
```

Comment:
IMO `how you format your message can look` or `formatting your message can look like` but not "how" & "like" together #nitpick